ydb/library/yql/parser/pg_wrapper/parser.cpp

```diff
@@ -216,6 +216,35 @@ void PGParse(const TString& input, IPGParseEvents& events) {
     }
 }
 
+List* PGGetStatements(const TString& input) {
```

Comment:
TArenaMemoryContext holds ownership of returned List*, so it's use after free

Please use PGParse method above